### DISCUSSION TITLE
Ask for help! ! !      Why can't I import the repository？

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51820] [Repositories] [answered]
### Select Topic Area

General

### Body

![tmpE784](https://user-images.githubusercontent.com/128476477/229473182-42d75df5-eca3-4d89-aa7f-cc3b857f6590.png)

I imported a project to the repository, but I was stuck until this interface could not be completed

**Comment by Fin1704** [comment]
I assume you are encountering an issue while trying to optimize your repository and push commits to GitHub. Here are a few steps you can try to resolve the issue:

Check your internet connection: Ensure that your internet connection is stable and strong enough to push your commits to GitHub. A weak or unstable internet connection can cause the push to hang or fail.

Check the size of your repository: If your repository is large, it may take longer to optimize and push to GitHub. In some cases, the optimization process may even time out or fail. If this is the case, you can try optimizing and pushing smaller portions of the repository at a time.

Verify your credentials: Make sure that you are authorized to push commits to the repository. Check your GitHub credentials to ensure that you have the necessary permissions to push to the repository.

Try pushing from a different machine: If none of the above steps work, you can try pushing your commits from a different machine or internet connection. This will help you determine if the issue is with your local environment or the repository itself.

**Reply by ghost** [reply]
thank you for your reply！！！
Your assumption is correct, I do have a problem here
After checking that my network is in good condition, the files in this repository are not large. I also tried to push from different machines and networks, but the results were not successful.
This repository is public so I have permission to operate.
At the same time, I use another account to operate successfully in the same environment
So, I think it might be my account problem, but I don't know what the problem is

**Reply by abbasl** [reply, marked as answer]
@sssukke Did your project finish importing? I'm having the same issue. The import process has been running ~20 hrs now.

**Reply by ghost** [reply]
I'm not,I use another account,now!

**Comment by imidg1825** [comment]
Test answer for Galaxy Brain
